docs/api/components/schemas.yaml

```diff
@@ -1915,6 +1915,14 @@ Database:
                   name:
                     description: The name of the role for which audit logging is disabled.
                     type: string
+    disable_public_all_docs:
+      description: |-
+        This controls whether the `_all_docs` REST API endpoint is publicly accessible or not.
```

Comment:
Nice to have but maybe add links:

[_all_docs](#operation/get_keyspace-_all_docs)

This link will resolve within the admin endpoint, but I think it is still helpful to see what the link does.